src/modules/events/send_event.h

```diff
@@ -48,55 +48,102 @@ namespace events
 
 extern "C" __EXPORT int send_event_main(int argc, char *argv[]);
 
+/** @class SendEvent The SendEvent class manages the RC loss audible alarm, LED status display, and thermal calibration. */
 class SendEvent : public ModuleBase<SendEvent>, public ModuleParams
 {
 public:
+	/** @brief Default Constructor. */
```

Comment:
Please remove. You have not provided arguments that justify adding redundant comments.

Reply:
It was worth a try to complete the docs. ;) Pushed up!  Thanks!